BUILD

```diff
@@ -537,7 +537,7 @@ cc_binary(
         "-DQDECL=",
         "-pthread",
     ],
-    linkopts = ["-pthread"],
+    linkopts = ["-pthread", "-lm"],
```

Comment:
Please keep each list item on a separate line. (See how the rest of the file is formatted.)